HangMan/HangMan2.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Apr  9 15:39:01 2020
+
+@author: cpantarotto
+"""
+# Lib
+import random
+#import numpy
+# DataSet of words
+global_dictionary = ["outside","squirrel", "straight", 
+              "finger", "leaf", "frog", "mouth", "grandson"]
+def hangmanGraphic (numberOfLives):
+    if numberOfLives==6:
+        print("_________")
+        print("|/   | ")
+        print("|")
+        print("|")
+        print("|")
+        print("|")
+        print("|")
+        print("|_______")            
+    if numberOfLives==5:
+        print("_________")
+        print("|/   | ")
+        print("|   (_)")
+        print("|")
+        print("|")
+        print("|")
+        print("|")
+        print("|_______")  
+    if numberOfLives==4:
+        print("_________")
+        print("|/   | ")
+        print("|   (_)")
+        print("|    |")
+        print("|    |")
+        print("|")
+        print("|")
+        print("|_______") 
+    if numberOfLives==3:
+        print("_________")
+        print("|/   | ")
+        print("|   (_)")
+        print("|   /|")
+        print("|    |")
+        print("|")
+        print("|")
+        print("|_______") 
+    if numberOfLives==2:
+        print("_________")
+        print("|/   | ")
+        print("|   (_)")
+        print("|   /|\\")
+        print("|    |")
+        print("|")
+        print("|")
+        print("|_______") 
+    if numberOfLives==1:
+        print("_________")
+        print("|/   | ")
+        print("|   (_)")
+        print("|   /|\\")
+        print("|    |")
+        print("|   / ")
+        print("|")
+        print("|_______") 
+    if numberOfLives==0:
+        print("_________")
+        print("|/   | ")
+        print("|   (_)")
+        print("|   /|\\")
+        print("|    |")
+        print("|   / \\")
+        print("|")
+        print("|_______") 
+
+
+def coolDisplay(word, known_bool):
+    cool_word = word[0]
+    #Build known_word | known_word = "h _ _ _ _ _ _"
+    for i in range (1, len(word), 1): #>1 as the first letter is already displayed
+        if known_bool[i]==0:
+            cool_word = cool_word + " _"
+        elif known_bool[i]==1:
+            cool_word = cool_word + " " + word[i]
+    return cool_word
+
+def main ():
+# Start of the program
+    lives = 6
```

Comment:
This one should be a global variable since it is a fixed configuration

Reply:
Done